funcs_utils.php

```diff
@@ -389,7 +394,7 @@ function setup_directories($input, $dirs = [DATA_DIR, MODULES_DIR]) {
 function filtered_modules($cmsMajor, $input)
 {
     $repos = MetaData::removeReposNotInCmsMajor(
-        MetaData::getAllRepositoryMetaData(false),
+        MetaData::getAllRepositoryMetaData(false, false),
```

Comment:
This allowed me to use the fork of supported modules in https://github.com/silverstripe/supported-modules/pull/53

There's no hosted scenario where standardiser is run, so it makes sense to use the local copy of the list going forward.

Reply:
I not sure about this change

Flip side of this is if someones 'reuses' an old local copy of module-standardiser and just run `git pull` on it to get the latest code, but forget to run `composer update` which seems like it would be an easy thing to do. Fetching from remote is more fail safe in that regard.

If you REALLY needed to do this for a one off e.g. a new module, you can just make this same code update locally and not commit it?

Maybe this would be better as an opt in command line option?

Reply:
Do you reuse an old copy or do you clone fresh? The two of us are the only ones using the tool so we can decide what the best practice is.
I personally re-clone it every time I use it, so that problem wouldn't affect me. If you also re-clone it every time, I suggest that we simply update the docs to say that's best practice and merge this.

If you do _not_ re-clone it every time, I'm okay with skipping out on this change.

Reply:
I'd say I reuse more than reclone, so we should skip this change IMO

Reply:
Reverted. Also just noticed this is being called twice in this function so reduced to a single call.